broadinstitute/single_cell_portal_core
# hardening study file id assignment (SCP-4101)

SCP-4101.  This generates mongo ids for new study files on the client side, which makes server<->client reconciliation much simpler.  This also contains some new logging of study file properties on saves to help troubleshoot the `invalid suffix` bug

TO TEST:
1. Go to upload wizard
2. create/update/delete a bunch of files, with a mix of slow (>10 seconds between) and fast (<3 seconds) between actions
3. confirm UI display consistently reflects the correct server state.
